docs/guides/getting_started/samples/intro/structure.cs

```diff
@@ -92,16 +99,16 @@ private static Task Logger(LogMessage message)
         // and other dependencies that your commands might need.
         _map.AddSingleton(new SomeServiceClass());
 
-        // Either search the program and add all Module classes that can be found:
-        await _commands.AddModulesAsync(Assembly.GetEntryAssembly());
-        // Or add Modules manually if you prefer to be a little more explicit:
-        await _commands.AddModuleAsync<SomeModule>();
-
         // When all your required services are in the collection, build the container.
         // Tip: There's an overload taking in a 'validateScopes' bool to make sure
         // you haven't made any mistakes in your dependency graph.
         _services = _map.BuildServiceProvider();
 
+        // Either search the program and add all Module classes that can be found:
```

Comment:
I'd suggest placing a comment here specifying that modules have to be public if they are to be automatically loaded.